Dockerfile

```diff
@@ -1,5 +1,6 @@
 FROM alpine:3.16.2
 RUN apk --no-cache add ca-certificates git
+RUN apk add --upgrade expat-dev
```

Comment:
Could you revert it? Those packages will be updated when the next version is released because Alpine tries to use the latest version at that time. I'll cut off v0.32.1, then those vulns will be addressed.

Reply:
@knqyf263 done